internal/service/organizations/organizational_unit_child_accounts_test.go

```diff
@@ -0,0 +1,51 @@
+package organizations_test
```

Comment:
Can you document how we run these? Is it just `go test`?

Reply:
There's some documentation [here](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/development-environment.md#testing-the-provider) on provider acceptance testing. Just under that section is instructions for using the provider locally for manual testing. That's how I did all my testing so far. The automated tests are tricky b/c you can't easily destroy the account resources after creating them so in practicality you get one run before you have to change the tests.